src/threadDb/hub-helpers.js

```diff
@@ -1,8 +1,8 @@
 const { createAPISig, Client, createUserAuth, PrivateKey } = require('@textile/hub');
```

Comment:
Let's move all the middleware related files outside the `src` in a `middleware` directory as it is not a part of the SDK.